examples/tutorial/variables.tf

```diff
@@ -129,7 +129,7 @@ variable "postgresql" {
     units   = optional(number, 1)
     trust   = optional(bool, true)
     config  = optional(map(string), {})
-    channel = optional(string, "14/stable")
+    channel = optional(string, "14/edge")
```

Comment:
question: why is this needed?

Reply:
it's due to https://github.com/canonical/postgresql-k8s-operator/issues/1011#issuecomment-3197943686